app/workers/email_bike_possibly_found_notification_worker.rb

```diff
@@ -1,24 +1,25 @@
 class EmailBikePossiblyFoundNotificationWorker < ScheduledWorker
+  prepend ScheduledWorkerRecorder
+
   def self.frequency
     24.hours
   end
 
   def perform
-    record_scheduler_started
-    notify_of_bike_index_held
-    record_scheduler_finished
+    Bike
```

Comment:
Similarly to the other worker, please separate this out into two things - one which finds the matches and one which sends messages to the matches.

Reason being - it's easier to test and it makes it more fault tolerant if a single message fails to send.

Reply:
Updated in f2fca2fa